src/internet_identity/tests/integration/anchor_management/last_usage_timestamp.rs

```diff
@@ -38,7 +38,7 @@ fn should_set_last_usage_on_add() -> Result<(), CallError> {
     )?;
 
     env.advance_time(Duration::from_secs(1));
-    let expected_timestamp = time(&env);
+    let expected_timestamp = time(&env) + 1; // +1 to account for the execution of the next call
```

Comment:
nit: it is slightly "ugly" that we have `+1` and the same comment on each `expected_timestamp`-computation.  Maybe it would be nicer to do `let expected_timestamp = time(&env);` after the next api call?

Reply:
Of course, thanks. 🤦